packages/jsdom-crawler/src/internals/jsdom-crawler.ts

```diff
@@ -274,9 +274,7 @@ export class JSDOMCrawler extends HttpCrawler<JSDOMCrawlingContext> {
     }
 
     override async _runRequestHandler(context: JSDOMCrawlingContext) {
-        context.parseWithCheerio = async () => {
-            return cheerio.load(context.body);
-        };
+        context.parseWithCheerio = () => Promise.resolve(cheerio.load(context.body));
```

Comment:
just to keep it consistent with CheerioCrawler
![Screen 2023-05-03 at 09 05 53](https://user-images.githubusercontent.com/697301/236777788-024ac093-6622-43b7-a653-edb48781a1ce.png)